### PR TITLE
Event Details Page Fixes

### LINKED ISSUE #2: [Component] Event Details Page 🎎
# Event Details Page

### How to navigate to this component
* Clicking on a marker on the map
* Clicking on the event on the Event List Dashboard

### Component purpose
* Shows event details

### LINKED ISSUE #42: [Bug] Explore Bar issue
**Describe the bug**
Explore bar looks wrong, search bar is too far down

**To Reproduce**
1. Go to the Discover page

**Expected behavior**
Middle explore button should be larger
Also want to add a 'shadow' above the explore bar to make it visible that its separate from the rest of the screen. White on white causes what can be seen below
Screen Shot 2020-03-10 at 12 22 14 AM

**Screenshots**
Screenshot (213)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Chromium Edge

### LINKED ISSUE #52: [Rework] Address changes in event details page
# Event Details Page

PR #49 

- [x] Working component

### PR DESCRIPTION
**Pull request by cadrianxyz**
# Description
Fixed requested changes on PR #49 

Fixes #2 #52 

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
~~Breaking change (fix or feature that would cause existing functionality to not work as expected)~~
~~This change requires a documentation update~~

# Checklist:

- [x] Code follows the project's style guidelines
- [x] I have performed a self-review of the Code being pushed
- [ ] Code has been reviewed by at least one other developer
- [x] Code has appropriate commenting, particularly in hard-to-understand areas
~~Corresponding changes to the documentation have been made~~
- [x] The changes generate no new warnings

### CODE REVIEW

**Comment by kraytaps** [comment]
wait should i review this or nah these r all my commits LOL

**Review by kraytaps — APPROVED** [review]
i approve

**Comment by cadrianxyz** [comment]
Um no I didn't assign reviewers.... lol. This shouldn't be reviewed unless you're done!
I made the branch a PR just to compare the new commits with master

The fact that its prefixed as a '**WIP**' too means that it shouldn't be reviewed or approved either hehe. [Deets here!](https://github.com/orgs/onlinkers/teams/principal)

**Comment by kraytaps** [comment]
@ad2969 i fixed all the changes requested and also fixed the explore bar
### Note
gallery / image viewer is not included

**Comment by kelvin-muk** [comment]
explore bar unfixed: all the categories clustered together

**Comment by cadrianxyz** [comment]
@kraytaps I wouldn't worry about the  _getting data_ part because I'm doing API integrations on a PR this week (and it works so far!) - would be easier to prevent complicated merge conflicts later on

**Comment by kraytaps** [comment]
@kelvin-muk explore bar should be ok now can u test it again
@ad2969 it's okay i think we can just merge this one first then we can resolve the conflicts later? i'll have to take a better look at your PR as well

**Comment by cadrianxyz** [comment]
- Limit tags to 4 (and add ovefflow)
- Make parallax work again (bye loading for now)

File: src/assets/js/eventData.js [hunk 1]
```diff
@@ -473,7 +479,8 @@ export default [
       admin: "UBC Events",
       description:
         "A family event for all ages, the UBC Apple Festival celebrates one of BC’s favourite fruits. From learning about the diversity of apples to tasting rare and unusual varieties, the Apple Festival is a great opportunity to discover more about this delicious fruit.",
-      rating: 4
+      rating: 4,
+      lightTheme: true
```
Comment (cadrianxyz) [inline src/assets/js/eventData.js]: > :warning: Highly illegal move.

Removing "lightTheme" - this is changing database schema and not recommended. @kraytaps In the future, things like this **NEED** to be communicated with the backend team before anything.

**Pull request merged**